doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -173,6 +184,10 @@ a. Download and extract the Service OS kernel
      .. note::
         You will also need to adjust the kernel name if you used a different
         RPM file as the source of your Service OS kernel.
+      
+      .. note::
+        The command line for the kernel in /etc/grub.d/40_custom should be all 
```

Comment:
The indenting is off for this note's contents.  It needs to align with the ``n`` in ``note``.